lib/shoulda/matchers/integrations/test_frameworks/rspec_expectations.rb

```diff
@@ -0,0 +1,34 @@
+module Shoulda
+  module Matchers
+    module Integrations
+      module TestFrameworks
+        # @private
+        class RspecExpectations
+          Integrations.register_test_framework(self, :rspec_exp)
+
+          def validate!
+            return if defined? RSpec::Matchers
+            raise TestFrameworkNotFound, <<-EOT
```

Comment:
Unless there's a need to explicitly support Ruby < 2.3, this would be nicer with a 'squiggly heredoc': https://infinum.co/the-capsized-eight/multiline-strings-ruby-2-3-0-the-squiggly-heredoc

Reply:
ok

Reply:
I'm fine with this -- this makes it easier to wrap the message at 80 characters.